api/contract.go

```diff
@@ -109,6 +109,19 @@ func (r *putFilterOutputRequest) Valid() error {
 	return nil
 }
 
+type AddOptionResponse struct {
```

Comment:
Why is this AddOptionResponse? Should it be GetFilterDimensionOptionsItem or FilterDimensionOption etc

Reply:
The swagger spec was defined as such `            $ref: '#/definitions/AddOptionResponse'` ... I was pretty confused by it, but left it as it was defined

Reply:
Please rename it, it's more important it's not confusing/is consistent in the code than keeping 100% to the name in the swagger spec. Update the name in the swagger spec to match the correct name if anything.